tests/components/charts.test.ts

```diff
@@ -328,12 +328,31 @@ describe("charts", () => {
     await simulateClick(".o-panel .inactive");
     await simulateClick(figures[1] as HTMLElement);
     await simulateClick(".o-chart-title input");
+    setInputValueAndTrigger(".o-chart-title input", "new_title", "input");
     setInputValueAndTrigger(".o-chart-title input", "new_title", "change");
 
     expect(model.getters.getChartDefinition("1").title).toBe("old_title_1");
     expect(model.getters.getChartDefinition("2").title).toBe("new_title");
   });
 
+  test.each(["basicChart", "scorecard", "gauge"])(
+    "edit chart title followed by clicking on grid will update the title of %s",
+    async (chartType: string) => {
+      createTestChart(chartType);
+      await nextTick();
+      await simulateClick(".o-figure");
+      await simulateClick(".o-figure-menu-item");
+      await simulateClick(".o-menu div[data-name='edit']");
+      await nextTick();
+      await simulateClick(".o-panel .inactive");
+      setInputValueAndTrigger(".o-chart-title input", "new_title", "input");
+      setInputValueAndTrigger(".o-chart-title input", "new_title", "change");
```

Comment:
In master, this specific line triggers the `updateTitle` method of the sidepanel which in turn updates the chart. This  means the test passes in master. you should remove it (which would make the test fail in master) and the see if your fix works